ts/utils/deepLink.ts

```diff
@@ -1,6 +1,8 @@
 import { Platform } from "react-native";
 import { NavigationNavigateActionPayload } from "react-navigation";
 
+export const ioItaliaLink = "https://io.italia.it/";
```

Comment:
It seems to be never used

Reply:
removed within https://github.com/pagopa/io-app/pull/1629/commits/da1bbdfd04c40079d66e00455e3705a3b9a5b0c9